src/index.ts

```diff
@@ -76,7 +50,7 @@ const parseInfo = (info?: string) => {
   );
 };
 
-const readLines = (fileContents: string) => {
+const readLines = async (fileContents: string) => {
```

Comment:
Not sure why this is async all of a sudden

Reply:
Had accidentally forgotten to remove it!